client/components/Tracker/index.jsx

```diff
@@ -19,9 +26,13 @@ const Tracker = () => {
 
         Banner
       </Banner>
-      <Form />
+      <Form getList={() => getList()} />
```

Comment:
`getList={getList}` can be used instead here, I believe.

Reply:
resolved Joseph will push changes. Thanks Jessica!